server/bin/test-bin/test-find-behavior

```diff
@@ -62,4 +62,5 @@ declare -A ub=(
 for bucket in small medium large huge; do
     printf "\nBucket: %s -- %s <= size < %s\n" "${bucket}" "${lb[${bucket}]}" "${ub[${bucket}]}"
     sort ${bucket}.lis
```

Comment:
why do we need to sort it before removing?

Reply:
Mostly because of how our unit tests work.  The tests are based on "gold" file comparisons, which means if the order of those buckets change in how they are processed, then the gold file comparisons will fail.

We sort all the buckets so that they maintain the same order independent of the environment in which the tests run.